src/internal/HandlerContext.ts

```diff
@@ -1,6 +1,10 @@
+import { Effect } from '../Effect'
 import { Fx } from '../Fx'
 
 export interface HandlerContext extends Fx<unknown, unknown> {
   readonly handlers: ReadonlyMap<unknown, (e: unknown) => Fx<unknown, unknown>>
-  readonly controls: ReadonlyMap<unknown, (resume: (a: any) => unknown, e: unknown) => Fx<unknown, unknown>>
```

Comment:
This wasn't needed anywhere, and since `control` handlers shouldn't move around, it's probably best to remove them from this interface until we have a real use case for them to be exposed.